src/components/App.tsx

```diff
@@ -19,35 +19,36 @@ export const App: React.FC = () => {
   const [activeTab, setActiveTab] = useState<"location" | "simulation">("location");
   const [latitude, setLatitude] = useState("");
   const [longitude, setLongitude] = useState("");
-  const [dayOfYear, setDayOfYear] = useState(171);
+  const [dayOfYear, /*setDayOfYear */] = useState(171);
```

Comment:
Not a change in this PR, but perhaps a comment indicating significance of `171`.